docs/aws/nRFConnectForCloudAPI.rst

```diff
@@ -0,0 +1,55 @@
+.. _aws-nrf-connect-for-cloud-api:
+
+nRF Connect for Cloud API
+#########################
+
+You can enable the :ref:`cell geolocation lookup <app-cellgeolocation>` for the application using `nRF Connect for Cloud's geolocation API <https://api.nrfcloud.com/v1/#operation/GetSingleCellLocation>`_  in your deployment.
+
+Creating an API device
+**********************
+
+The nRF Connect for Cloud API requires for some calls a device identifier.
+It is recommended to create a new device for the API operations.
+
+The following commands create a new device using `httpie <https://httpie.io/>`_:
```

Comment:
```suggestion
Create a new device using `httpie <https://httpie.io/>`_:
```
You might also want to make these numbered steps. Meaning: 1. Create a new device 2. Connect the device 3. Associate the device with your account

Reply:
So you did not like the idea of having the numbered steps? I know the heading only says creating and that is the only step in that sense, but you still have other  advice for connecting and associating the device.

Reply:
No, I don't think numbers add anything meaningful to this document.
And we have other places which have sequential instructions that are not numbered, which I think is perfectly fine: https://nordicsemiconductor.github.io/asset-tracker-cloud-docs/saga/docs/aws/GettingStarted/DeviceCredentials.html

Reply:
@umapraseeda what do you think?

Reply:
@coderbyheart in the example you gave, the steps are separate sections, therefore, not numbered. I would argue that numbered steps help when you have procedures you need to do in a certain order. Although, this is a short procedure with three steps.

Reply:
I suggest having numbered steps here as the steps are sequential.

Reply:
@umapraseeda please check the latest version of the PR, the creation of the device has been removed.